src/vs/workbench/contrib/notebook/browser/contrib/outline/notebookOutline.ts

```diff
@@ -303,12 +305,48 @@ export class NotebookCellOutline implements IOutline<OutlineEntry> {
 		if (!widget) {
 			return Disposable.None;
 		}
-		widget.revealInCenterIfOutsideViewport(entry.cell);
+
+		if (entry.range) {
```

Comment:
We can lift IRange first by `const range = Range.lift(entry.range)`

Reply:
ah, nice. Curious why Range just have a ctor that takes an IRange?

Reply:
I think it makes sense to have a ctor override which takes in IRange.